gibbon-compiler/examples/imports/failing.md

```diff
@@ -0,0 +1,61 @@
+# Poly
```

Comment:
This file is also a layout-patch leftover?

Reply:
this was leftover notes for myself, good to remove from the diff